src/SurveyService.Data/OptionRepository.cs

```diff
@@ -60,4 +66,19 @@ public Task DisactivateAsync(ICollection<DbOption> options)
 
     return _provider.SaveAsync();
   }
+
+  public async Task<bool> EditAsync(JsonPatchDocument<DbOption> patch, DbOption dbOption)
+  {
+    if (patch is null || dbOption is null)
+    {
+      return false;
+    }
+    
+    patch.ApplyTo(dbOption);
+    dbOption.ModifiedBy = _httpContextAccessor.HttpContext.GetUserId();
```

Comment:
add to method's arguments: Guid? modifiedBy=null
line 78: dbOption.ModifiedBy =modifiedBy;
and in command:
.... _optionRepository.EditAsync(_mapper.Map(patch), dbOption, requestSenderId));

Reply:
Done